src/common/mfu_flist_copy.c

```diff
@@ -170,8 +197,7 @@ static int mfu_copy_open_file(
         } else {
             /* the file we're trying to open is different,
              * close the old file and delete the name */
-            mfu_file_close(name, mfu_file);
-            mfu_free(&cache->name);
+            mfu_copy_close_file(cache, mfu_file);
```

Comment:
Is this change part of fixing the nsec bug?

Reply:
no it was to make sure the sync is called with the close of file. It will be good for other FS where the sync is enabled on the backend.

Reply:
That change was from Adam but never merged.

Reply:
I'll take it out and add a new MR as it is a different issue.